src/urllib3/contrib/emscripten/fetch.py

```diff
@@ -322,6 +432,17 @@ def is_worker_available() -> bool:
 
 
 def send_streaming_request(request: EmscriptenRequest) -> EmscriptenResponse | None:
+    if has_jspi():
+        return send_jspi_request(request, True)
+    elif is_in_node():
+        raise _RequestError(
+            message="urllib3 only works in node.js with pyodide.runPythonAsync"
+            " and requires the flag --experimental-wasm-stack-switching in "
+            " versions of node <24",
```

Comment:
We may want to mention here that node >= 20 is required. We're not sure that node 24 won't require the flag yet...

Reply:
done